src/test/java/ancaMarian/tests/tema5/DemoQA.java

```diff
@@ -0,0 +1,212 @@
+        firstName.sendKeys("Ana");
+    }
+
+    public static void addLastName() {
+        WebElement lastName = driver.findElement(By.id("lastName"));
+        lastName.click();
+        lastName.clear();
+        lastName.sendKeys("Maria");
+    }
+
+    public static void addEmail() {
+        WebElement email = driver.findElement(By.xpath("//*[@id='userEmail']"));
+        email.click();
+        email.clear();
+        email.sendKeys("anamaria@gmail.com");
+    }
+
+    public static void addAge() {
+        WebElement age = driver.findElement(By.xpath("//*[@id='age']"));
+        age.click();
+        age.clear();
+        age.sendKeys("22");
+    }
+
+    public static void addSalary() {
+        WebElement salary = driver.findElement(By.xpath("//*[@id='salary']"));
+        salary.click();
+        salary.clear();
+        salary.sendKeys("2200");
+    }
+
+    public static void addDepartment() {
+        WebElement department = driver.findElement(By.xpath("//*[@id='department']"));
+        department.click();
+        department.clear();
+        department.sendKeys("boss");
+    }
+
+    public static void clickSubmitButton() {
+        WebElement submitButton = driver.findElement(By.xpath("//*[@id='submit']"));
+        submitButton.click();
+    }
+
+    public static void useSearchBar() {
+        WebElement searchBar = driver.findElement(By.id("searchBox"));
+        searchBar.click();
+        searchBar.clear();
+        searchBar.sendKeys("ana");
+        searchBar.clear();
+    }
+
+    public static void edit() {
+        WebElement editButton = driver.findElement(By.cssSelector("#edit-record-4 > svg"));
+        editButton.click();
+        WebElement departmentToEdit = driver.findElement(By.id("department"));
+        departmentToEdit.click();
+        departmentToEdit.clear();
+        departmentToEdit.sendKeys("intern");
+        WebElement submitButtonInEdit = driver.findElement(By.cssSelector("#submit"));
+        submitButtonInEdit.click();
+    }
+
+    public static void delete() {
+        WebElement deleteRecord = driver.findElement(By.id("delete-record-4"));
+        deleteRecord.click();
+    }
+
+    public static void buttons() {
+        WebElement buttons = driver.findElement(By.id("item-4"));
+        buttons.click();
+    }
+
+    public static void doubleClick() {
+        WebElement doubleClickButton = driver.findElement(By.id("doubleClickBtn"));
+        Actions actionDoubleClick = new Actions(driver);
+        actionDoubleClick.doubleClick(doubleClickButton).perform();
+    }
+
+    public static void rightClick(){
+        WebElement rightClickButton = driver.findElement(By.id("rightClickBtn"));
+        Actions rightClick = new Actions(driver);
+        rightClick.contextClick(rightClickButton).perform();
+    }
+
+    public static void clickButton(){
+        WebElement clickButton = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[2]/div[2]/div[3]/button"));
```

Comment:
You can use "//button[text()='Click Me']" instead of "/html/body/div[2]/div/div/div[2]/div[2]/div[2]/div[3]/button".

Reply:
thank you for the suggestion!